internal/eval/worker_eval.go

```diff
@@ -6,28 +6,10 @@ package eval
 // allowing them to process commands and return appropriate responses.
 
 import (
-	"bytes"
-	"fmt"
-
 	"github.com/dicedb/dice/internal/clientio"
 	diceerrors "github.com/dicedb/dice/internal/errors"
-	dstore "github.com/dicedb/dice/internal/store"
 )
 
-// RespINFO generates a response for the INFO command.
-// The INFO command provides the total number of keys per database in the keyspace.
-// It creates a buffer that stores the keyspace information and formats it according to the Redis protocol.
-// The function returns the encoded buffer as the response.
-func RespINFO(args []string) []byte {
-	var info []byte                      // Initialize a byte slice to hold the keyspace info.
-	buf := bytes.NewBuffer(info)         // Create a buffer for efficiently writing the response.
-	buf.WriteString("# Keyspace\r\n")    // Write the header "# Keyspace" to indicate the section for keyspace stats.
-	for i := range dstore.KeyspaceStat { // Iterate through the KeyspaceStat map, which holds the number of keys per database.
-		fmt.Fprintf(buf, "db%d:keys=%d,expires=0,avg_ttl=0\r\n", i, dstore.KeyspaceStat[i]["keys"]) // Format the keyspace info for each database.
-	}
-	return clientio.Encode(buf.String(), false) // Encode the result into a Redis protocol-compliant format and return the encoded bytes.
-}
```

Comment:
Just for my understanding, do we not need this anymore?

Reply:
This should still be here. However, since this becomes a multi-shard command, maybe that's why he's removing it for now, is my guess.

Reply:
Yes, I had removed this as it is a multi-key command...